doc/WebSite/Abp-Session.md

```diff
@@ -46,34 +46,34 @@ AbpSession defines a few key properties:
 -   **TenantId**: Id of the current tenant or null if there is no
     current tenant (in case of user has not logged in or he is a host
     user).
--   **ImpersonatorUserId**: Id of the impersonator user if current
+-   **ImpersonatorUserId**: Id of the impersonated user if the current
```

Comment:
This is Id of the user who performed the impersonation operation on behalf of the current user. Assume that my user id is 1 and I performed impersonation for the user 2. In that case ImpersonatorUserId is 1 and UserId = 2. So, isn't impersonated user is 2?

Reply:
Not sure what you mean here.  Are you suggesting this addon to the documentation?

Reply:
Change it back to impersonator?

Reply:
"Id of the user impersonating, if the current
    session is impersonated by another user."

Reply:
Or.  

"Id of the impersonating user, if the current
    session is impersonated by another user."

Reply:
The original phrasing is more straightforward.

Reply:
Fixed